includes/class-demo-importer.php

```diff
@@ -191,7 +191,7 @@ public function enqueue_scripts() {
 							'<ol><li class="warning">' . __( 'Importing the demo on the site if you have already added the content is highly discouraged.', 'themegrill-demo-importer' ) . '</li>',
 							'<li>' . __( 'You need to import demo on fresh WordPress install to exactly replicate the theme demo.', 'themegrill-demo-importer' ) . '</li>',
 							'<li>' . __( 'It will install the required plugins as well as activate them for installing the required theme demo within your site.', 'themegrill-demo-importer' ) . '</li>',
-							'<li>' . __( 'Copyright images will get replaced with other placeholder images.', 'themegrill-demo-importer' ) . '</li>',
+							'<li>' . __( 'Please remember to replace all demo images with your own to avoid copyright infringement.', 'themegrill-demo-importer' ) . '</li>',
```

Comment:
Suggestion:

To avoid copyright infringement, ensure to replace all demo images with your own after importing the demo.